ivy/functional/frontends/numpy/statistics/averages_and_variances.py

```diff
@@ -141,38 +141,42 @@ def std(
     return ret.astype(dtype, copy=False)
 
 
+def weighted_average(
```

Comment:
```suggestion
def _weighted_average(
```
Since it's a private helper function, we should append the function name with "_".